app/models/merchant.rb

```diff
@@ -1,4 +1,30 @@
 class Merchant < ApplicationRecord
   has_many :invoices
   has_many :items
+
+  has_many :customers, through: :invoices
+  has_many :transactions, through: :invoices
+  has_many :invoice_items, through: :invoices
+
+  def self.customers_with_pending_invoices
+    joins(:invoices, :customer).where(invoice:{status:"pending"})
```

Comment:
Take another look at the definition of a pending invoice based on the spec. I don't think this is going to get you there.